rhobs/alerting/control_plane/prometheus.argocd_alerts.yaml

```diff
@@ -24,6 +24,7 @@ spec:
           Application: {{ $labels.name }}
           Cluster: {{ $labels.dest_server }}
           Health status Degraded.
+        namespace: '{{ $labels.dest_namespace }}'
```

Comment:
Maybe we can name this annotation differently somehow? Currently it will mean that there's a namespace label and a namespace annotation with different values, which might be confusing.

Maybe can give it a name that will suggest what it's for. e.g. `alert_routing_namespace` or something along those lines).

Reply:
WDYT about `alert_namespace_tag` or `alert_namespace_route`?

Reply:
I think it should end with `namespace`, and whatever comes before it should describe the namespace. Can be `alert_route_namespace`.

Reply:
changed